CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 # Unreleased
+### Changed
```

Comment:
```suggestion
## Changed
```

Reply:
I would like to keep this change until we fix the format of CHANGELOG.md